zproject/urls.py

```diff
@@ -335,6 +340,7 @@
     rest_path("messages/flags", POST=update_message_flags),
     rest_path("messages/<int:message_id>/history", GET=get_message_edit_history),
     rest_path("messages/matches_narrow", GET=messages_in_narrow_backend),
+    rest_path("messages/report/<int:message_id>", POST=report_message_backend),
```

Comment:
I think the traditional REST URL for this would be `messages/<int:message_id>/report` -- probably worth switching to that for consistency.

Reply:
DONE.